x11-libs/gdk-pixbuf-loader-webp/gdk-pixbuf-loader-webp-0_p20191003.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils meson multilib-minimal
+
+MY_PN="webp-pixbuf-loader"
+
+DESCRIPTION="WebP Image format GdkPixbuf loader"
+HOMEPAGE="https://github.com/aruiz/webp-pixbuf-loader"
+COMMIT="fb04954d48425dfa9d7014e733736c1802ba9733"
+SRC_URI="https://github.com/aruiz/${MY_PN}/archive/${COMMIT}.tar.gz -> ${PN}-${PV}.tar.gz"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
```

Comment:
where are all these keywords coming from? who tested those?

Reply:
from original package

Reply:
I don't see that:

```
Keywords for x11-libs/gdk-pixbuf-loader-webp:
               |                               |   u   |  
               | a   a     p           s a r   |   n   |  
               | m   r i   p   h m s   p l i m | e u s | r
               | d a m a p c x p 6 3   a p s i | a s l | e
               | 6 r 6 6 p 6 8 p 8 9 s r h c p | p e o | p
               | 4 m 4 4 c 4 6 a k 0 h c a v s | i d t | o
---------------+-------------------------------+-------+-------
20160328234507 | o o o o o o o o o o o o o o o | 6 o 0 | gentoo
```